api/lua/Cargo.toml

```
@@ -0,0 +1,80 @@
+[package]
+name = "playdate-lua"
+version = "0.1.0"
+readme = "README.md"
+description = "High-level Lua API built on-top of Playdate API"
+keywords = ["playdate", "sdk", "api", "gamedev"]
+categories = ["game-development", "api-bindings", "no-std"]
+edition.workspace = true
+license.workspace = true
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+
+
+[features]
+default = ["sys/default"]
+
+# playdate-sys features, should be shared because it's build configuration:
+
+bindgen-runtime = ["sys/bindgen-runtime"]
+bindgen-static = ["sys/bindgen-static"]
+bindings-derive-debug = ["sys/bindings-derive-debug"]
+
+
+[dependencies]
+sys = { workspace = true, default-features = false }
+
+[dev-dependencies]
+system = { workspace = true, default-features = false, features = [ "try-trait-v2" ] }
+
+
+[[example]]
+name = "add-function-get-arg-string"
+crate-type = ["dylib", "staticlib"]
+path = "examples/add-function-get-arg-string.rs"
+required-features = ["sys/entry-point", "sys/lang-items"]
+
+
+[package.metadata.playdate]
+bundle-id = "rs.playdate.lua"
+
+[package.metadata.playdate.dev-assets]
+# The Lua runtime expects us to provide a main.pdz file at the root.
+#
+# Compiled with `pdc --skip-unknown --strip sources/main.lua Example.pdx`
+
+# This results in target/playdate/assets/playdate-lua-244d63d426e998ae/dev/build/main.pdz/examples/Example.pdx/main.pdz (not what we want)
+# "main.pdz" = "examples/Example.pdx/main.pdz"
+
+# This results in target/playdate/assets/playdate-lua-244d63d426e998ae/dev/build/examples/Example.pdx/main.pdz (not what we want)
+"/" = "examples/Example.pdx/main.pdz"
+
+# This requires moving main.pdz to api/lua/main.pdz
+# This results in target/playdate/assets/playdate-lua-244d63d426e998ae/dev/build/main.pdz/main.pdz (not what we want)
+# "main.pdz" = "main.pdz"
+
+# This requires moving main.pdz to api/lua/main.pdz
+# This results in target/playdate/assets/playdate-lua-244d63d426e998ae/dev/build/main.pdz (what we want)
+# "/" = "main.pdz"
```

@boozook is there a bug here or am I not understanding how this is supposed to work?

---

I thought maybe `method = "copy"` might work differently but `package.metadata.playdate.dev-assets.options` and `package.metadata.playdate.options.dev-assets` don't seem to be supported.

---

I fixed it finally! 🎉 (#339)
Plz use 
```toml
[package.metadata.playdate.dev-assets]
"main.pdz" = "examples/Example.pdx/main.pdz"
# or "/main.pdz" = "examples/Example.pdx/main.pdz" # (`/` is doesn't matter in this case)
```

---

```suggestion
"main.pdz" = "examples/Example.pdx/main.pdz"
```

Wow 😲 I learned how suggestions works! It works!